llvm/test/Feature/Repo/repo_hash_contributions_func.ll

```diff
@@ -0,0 +1,55 @@
+; This test is used to check the hash calculation for the contributions.
+;
+; All GOs' information are shown below:
+; GO Name: Var1, Initial Dependencies: [ Func1 ], Contributions: [ ]
+; GO Name: Var2, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Func1, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Main, Initial Dependencies: [ Func2 ], Contributions: [ Var2 ]
+; GO Name: Func2, Initial Dependencies: [ ], Contributions: [ Var1 ]
```

Comment:
This might be easier to read if it were formatted like a table.

Reply:
Changed to the table in commit cca1f2d.